src/library_postprocessing.rs

```diff
@@ -130,6 +130,17 @@ impl Library {
         if glib_name.is_none() { return None; }
         let glib_name = glib_name.unwrap();
         let detected_c_type = match *type_ {
+            Alias(ref alias) => {
+                if let Some(aliased) = self.c_type_by_type_id(alias.typ) {
+                    if aliased.find("*").is_some() {
+                        format!("{}*", glib_name)
```

Comment:
I can extract from this function `fn is_pointer(...)->bool` to make logic clearer